build.gradle.kts

```diff
@@ -21,6 +21,13 @@ repositories {
     maven("https://repo.papermc.io/repository/maven-public/")
     maven("https://jitpack.io")
     maven("https://repo.william278.net/releases")
+    maven {
+        url = uri("https://maven.pkg.github.com/Euphillya/Skyllia")
+        credentials(PasswordCredentials::class) {
+            username = System.getenv("MAVEN_USERNAME") ?: (project.findProperty("oraxenUsername") as? String ?: "defaultUsername")
+            password = System.getenv("MAVEN_PASSWORD") ?: (project.findProperty("oraxenPassword") as? String ?: "defaultPassword")
+        }
+    }
```

Comment:
not gonna add a repo that requires credentials

Reply:
I don't have any repositories other than github

Reply:
then i wont be able to add this sadly
You can add compatibility in the plugin itself, doesnt NEED to be in ProtectionLib
Just cancel the events in question